appveyor.yml

```diff
@@ -5,4 +5,7 @@ test: off
 deploy: off
```

Comment:
we don't need deploy.
Also deploy isn't actually a `on`/`off` switch, but rather a list of different providers to deploy to (as well as their settings), or a script.

But all is handled in the Cake script instead, after a release have been published.

Reply:
right, we should probably just remove it. [Cake.Recipe](https://github.com/cake-contrib/Cake.Recipe/blob/develop/.appveyor.yml) does not have it in their yaml file. Other than that, go ahead and merge it!

Reply:
it can be removed, but there isn't any harm for having it there either.
Now, it just disables the check for any deployments to run.
But I can remove it at the same time as I merge this one, but working on a few other stuff at the moment so it may take some time.